src/gpu/d3d12/SDL_gpu_d3d12.c

```diff
@@ -6849,7 +6849,11 @@ static D3D12Fence *D3D12_INTERNAL_AcquireFence(
             return NULL;
         }
         fence->handle = handle;
-        fence->event = CreateEventEx(NULL, 0, 0, EVENT_ALL_ACCESS);
+#if defined(SDL_PLATFORM_GDK) // CreateEventEx() arrived in Vista, so we need an #ifdef for XP.
+        fence->event = CreateEventEx(NULL, NULL, 0, EVENT_ALL_ACCESS);
+#else
+        fence->event = CreateEventW(NULL, 0, 0, NULL);
+#endif
```

Comment:
Since we're passing `NULL` to the latest string input, I guess it does not matter much whether we're using `CreateEventA` or `CreateEventW`?
```suggestion
        fence->event = CreateEvent(NULL, 0, 0, NULL);
```